packages/utils/src/types/actions.ts

```diff
@@ -298,29 +298,29 @@ export type CreateActionDetail = z.infer<typeof CreateActionDetailSchema>
 COMPLETE
 */
 export type CompleteActionParams = {
+  completeAfter?: number
   chainId?: string
   boostId?: string
   actionType?: string
 }
 
-export const CompleteActionDetailSchema = z
-  .object({
-    chainId: z.string().optional(),
-    boostId: z.string().optional(),
-    actionType: z.string().optional(),
-  })
-  .refine(
-    (data) =>
-      data.chainId != null || data.boostId != null || data.actionType != null,
-    {
-      message:
-        'At least one of chainId, boostId, or actionType must be provided',
-    },
-  )
+export const CompleteActionDetailSchema = z.object({
```

Comment:
We can rely on this export once the utils are cleaned up vs copying it over the backend. No action needed mostly a reminder for me to go back and clean this up in the RH repo once that change is live.